platform/macos/detect.py

```diff
@@ -120,10 +120,10 @@ def configure(env: "Environment"):
 
     # CPU architecture.
     if env["arch"] == "arm64":
-        print("Building for macOS 11.0+.")
-        env.Append(ASFLAGS=["-arch", "arm64", "-mmacosx-version-min=11.0"])
-        env.Append(CCFLAGS=["-arch", "arm64", "-mmacosx-version-min=11.0"])
-        env.Append(LINKFLAGS=["-arch", "arm64", "-mmacosx-version-min=11.0"])
+        print("Building for macOS 13.5+.")
+        env.Append(ASFLAGS=["-arch", "arm64", "-mmacosx-version-min=13.5"])
+        env.Append(CCFLAGS=["-arch", "arm64", "-mmacosx-version-min=13.5"])
+        env.Append(LINKFLAGS=["-arch", "arm64", "-mmacosx-version-min=13.5"])
```

Comment:
Same here, it absolutely should stay at `11.0`, and use `@available` for code that require higher version.

Reply:
No worries - will update the PR back to macOS 11 for arm64.

Note that I don't have any way to test on macOS 11

Reply:
Me neither (at least in terms of real hardware), but I'll try testing it on Monterey VM later (but I'm not sure what are Metal pass-through driver capabilities).

Reply:
Done 👍🏻